lib/oban/queue/executor.ex

```diff
@@ -219,24 +219,7 @@ defmodule Oban.Queue.Executor do
         %{exec | result: result, state: :snoozed, snooze: seconds}
 
       returned ->
-        Logger.warn(fn ->
```

Comment:
This `case` is long enough to extract this block into a separate helper function for readability purposes and handle the case when we don't want to log a warning because we're running on _unsafe_ mode. Please let me know if you're ok with it.

Reply:
Totally ok with it 👍